odoo/tools/image.py

```diff
@@ -292,6 +295,53 @@ def image_process(base64_source, size=(0, 0), verify_resolution=False, quality=0
 # Misc image tools
 # ---------------------------------------
 
+def _encode_name(name):
```

Comment:
Probably now that we have the mixin we should move all of those to the mixin file. They are not really generic tools, mostly just useful method for the placeholder feature.